app/bundles/FormBundle/Translations/en_US/messages.ini

```
@@ -200,10 +200,14 @@ mautic.form.table.most.submitted="Most submitted forms"
 mautic.form.table.top.referrers="Top referrers"
 mautic.form.type.header="What type of form do you want to create?"
 mautic.form.type.campaign="Campaign Forms"
-mautic.form.type.campaign.description="Campaign forms can be selected to be the source of contacts when building a campaign. Some submit actions may not be available."
+mautic.form.type.one.differences="<li>Used within campaigns for contact sourcing</li>
```

```suggestion
mautic.form.type.one.differences="<li>Used to initiate automated campaign workflows</li>
```

I am not sure folks would understand what we mean by 'contact sourcing'. Maybe this fits better as it also explains that the form itself initiates the campaign workflow?

---

Totally makes sense